Sources/_StringProcessing/ByteCodeGen.swift

```diff
@@ -365,19 +427,18 @@ fileprivate extension Compiler.ByteCodeGen {
 
     builder.label(success)
   }
-  
+
   mutating func emitLookaround(
     _ kind: (forwards: Bool, positive: Bool),
     _ child: DSLTree.Node
   ) throws {
-    guard kind.forwards else {
-      throw Unsupported("backwards assertions")
-    }
+    reverse = !kind.forwards
     if kind.positive {
-      try emitPositiveLookahead(child)
+      try emitPositiveLookaround(child)
     } else {
-      try emitNegativeLookahead(child)
+      try emitNegativeLookaround(child)
     }
+    reverse = false
```

Comment:
Are nested lookarounds possible? If so, do we need to restore the prior reverse value instead of clearing it?

Reply:
I haven't tried yet but I suspect with this implementation they are not. Seems like a good thing to support though